<?xml version="1.0"?>
<launch>

  <node name="behavior_tree" pkg="ca_behavior_tree" type="node"
        output="screen" respawn="false">
    <param name="xml_tree" value="$(find ca_behavior_tree)/config/robomate.xml"/>
  </node>

</launch>
